Integrations/Tenable_sc/Tenable_sc.py

```diff
@@ -1293,8 +1306,7 @@ def get_device_command():
         'DNSName': device.get('dnsName'),
         'OS': re.sub('<[^<]+?>', ' ', device['os']).lstrip() if device.get('os') else '',
         'OsCPE': device.get('osCPE'),
-        'LastScan': datetime.utcfromtimestamp(int(device.get('lastScan'))).strftime(
-            '%Y-%m-%dT%H:%M:%SZ') if device.get('lastScan') and int(device['lastScan']) > 0 else '',
+        'LastScan': timestamp_to_utc(device['lastScan'], additional_cond=(int(device['lastScan']) > 0)),
```

Comment:
Could you please explain when is this "additional_cond" not true, and why?
I think there might be a more graceful and readable solution here